mantidproject/mantid
# Fix passing of args with spaces in them

MantidPlot, mantidpython and the workbench fails to load files from the argument list if it has spaces in them.

**To test:**
* Create a filename with spaces in them _e.g._ `echo "print('hello world')" >> file\ with\ spaces.py`
* Try starting MantidPlot, mantidpython and the workbench from the launch scripts and see if they now run
    * `bin/launch_mantidworkbench.sh -x ~/file\ with\ spaces.py`
    * `bin/launch_mantidplot.sh -x ~/file\ with\ spaces.py`
    * `bin/mantidpython ~/file\ with\ spaces.py`
    * `bin/mantidpython -i ~/file\ with\ spaces.py`
    * `bin/mantidpython --classic ~/file\ with\ spaces.py`

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
